beetmoverscript/newsfragments/1463749.feature

```diff
@@ -0,0 +1 @@
+Add support for language pack checksums.
```

Comment:
@Callek towncrier has a limited amount of operations that it supports. Can you please s,feature,added this file?